piety-promoted-v3/modernized/01-chapter-01.adoc

```diff
@@ -38,7 +38,7 @@ that you and yours may mind your future state above all,
 and let not the hurry of this transitory world, with all its tinsel glare, pride,
 grandeur, and vanity, choke the good seed, which, as it is permitted to take root,
 will bring forth the good fruit, which will entitle you to discipleship,
-and give you a beauty and glory which all these things cannot give.`'
+and give you a beauty and glory which all these things cannot give.
```

Comment:
it's a bit borderline as well, but let's embed the 2-paragraph letter right below as a letter. maybe a good rule of thumb would be, 2 or more paragraphs of a letter: go for an embed.

Reply:
Sounds good. 👍